apps/designer/components/Buttons/NewButton.tsx

```diff
@@ -0,0 +1,18 @@
+const NewButton = ({
```

Comment:
Better more descriptive names needed. Some componets probably already have a decent name given in figma file. Otherwise, try to give a name that is more specific than `NewButton`